VespaGAS/Models/Volume.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+enum VolumeUnit: CustomStringConvertible {
```

Comment:
haha, it doesn't this time no :) I was trying to think I how I would do this but I'm not sure, it clearly works well!